views/pulsar/v2/helpers/tabs.html.twig

```diff
@@ -29,9 +29,10 @@
             {% set options = ({}) %}
             <li class="nav-inline__item{% if tab_active is defined and tab_active == tab.id %} nav-inline__item--is-active is-active{% endif %}{% if tab.hidden is defined and tab.hidden == true %} hide{% endif %}">
                 {% if tab.disabled is defined and tab.disabled == true %}
-                    {% set href = '#' %}
+                    {% set href = '#' ~ tab.id %}
                     {% set type = 'link' %}
-                    {% set tabindex = '-1' %}
+                    {% set tabindex = '0' %}
```

Comment:
is a tabindex required here, wouldn't we arrive at it at the right time without it?

Reply:
Yeah I spotted this after pushing, already sorted :)